radio/src/targets/nv14/audio_spi_driver.cpp

```diff
@@ -355,6 +357,8 @@ void audioSendRiffHeader()
   audioSpiWriteBuffer(RiffHeader, sizeof(RiffHeader));
 }
 
+#if defined(PCBX12S) || defined(PCBNV14)
```

Comment:
Isn't this a NV14-specific driver? Just wondering why there are `#if defined(PCBX12S)` headers... maybe some cleanup is needed?

Reply:
Ther is a bunch of those scattered around in the NV14 code. Obviously someone either planned to merge this stuff together or did no bother to do a clean up after copying the files for a new target.

Reply:
I suspect the latter... think this is all basically from the openflysky fork of opentx for the nv14, and the stuff was probably just copied and hammered into shape and then left alone.

Reply:
Maybe open a new ticket for this just to track that some day it will require cleanup

Reply:
Well, what I tried to do is to use as much code from horus audio spi driver as possible to do this PR, so that we can do a merge in next release.  I do not want to change anything in horus related target, because I want to have this merge in 2.8.0 as a quick fix.  I will work out a merge after 2,8,0 release.

Reply:
For other NV14 specific drivers, I will take a look and see what can be merged in next release.  Well, I have quite familiar with all these codes and will try to unify them based on Raphael's new drivers.